sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/util/QueryChargeHelper.java

```diff
@@ -0,0 +1,20 @@
+package com.azure.digitaltwins.core.util;
+
+import com.azure.core.http.rest.PagedResponse;
+
+import java.util.Objects;
+
+/**
+ * A helper to extract the query charge from the query response.
```

Comment:
Could you add a note about what a query charge is?

Reply:
sure will do